quinn-proto/src/connection/mod.rs

```diff
@@ -5188,10 +5244,27 @@ pub enum PathError {
     ServerSideNotAllowed,
     /// Current limits do not allow us to open more paths
     MaxPathIdReached,
-    /// No remove CIDs avaiable to open a new path
+    /// No remote CIDs avaiable to open a new path
     RemoteCidsExhausted,
 }
 
+/// Errors triggered when abandoning a path
+#[derive(Debug, Error, Clone, Eq, PartialEq)]
+pub enum ClosePathError {
+    /// The path is already closed or was never opened
+    #[error("closed path")]
+    ClosedPath,
+    /// This is the last path, which can not be abandoned
+    #[error("last open path")]
+    LastOpenPath,
+}
+
+#[derive(Debug, Error, Clone, Copy)]
+#[error("Multipath extension not negotiated")]
+pub struct MultipathNotNegotiated {
+    _private: (),
+}
```

Comment:
why not just a unit-like struct
```suggestion
pub struct MultipathNotNegotiated;
```